e2e/cloud_manager/agent_api_keys_test.go

```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//go:build e2e || (cloudmanager && remote && replica) || (opsmanager && remote && replica)
+//go:build e2e || (cloudmanager && remote && replica) || (om44 && remote && replica) || (om50 && remote && replica)
```

Comment:
I am not sure if we want something like this 
```suggestion
//go:build e2e ||  remote && replica
```
or we want to keep it explicit who is running this test